modules/MapGenerator/AllRelation.php

```diff
@@ -1,42 +1,43 @@
+
 <?php
+/*
+ * @Author: Edmond Kacaj
+ * @Date: 2018-09-11 11:37:45
+ * @Last Modified by: edmondikacaj@gmail.com
+ * @Last Modified time: 2018-09-11 11:38:51
+ */
```

Comment:
This is not a License, this is you signing a file and adding some information that is totally redundant. git is already tracking who is making the change, when and the whole history MUCH better than you will ever do trying to maintain this information.

You don't have to sign your files, you have to commit a lot. If you absolutely must sign them, just put your name, all the dates and other stuff is just noise and put your name right after a well-defined license so people will know how they can use your code.

Reply:
Is something what Visual Code generate 
i need to check all file and to remove ??

Reply:
You can configure your IDE to not add that.
Yes, please change those.